src/current/_includes/releases/v23.2/v23.2.5.md

```diff
@@ -0,0 +1,68 @@
+## v23.2.5
+
+Release Date: May 7, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-5-sql-language-changes">SQL language changes</h3>
+
+- The new [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` enables collection of table statistics on virtual computed columns. [#120923][#120923]
+- The new [session variable]({% link {{ page.version.version}}/session-variables.md %}) `optimizer_use_virtual_computed_column_stats` configures the [optimizer]({% {{ page.version.version }}/make-queries-fast.md %}) to consider table statistics on virtual computed columns. [#121179][#121179]
+- The new `FORCE_INVERTED_INDEX` [hint]({% link {{ page.version.version }}/indexes.md %}#selection) configures the [optimizer]({% {{ page.version.version }}/make-queries-fast.md %})  to prefer a query plan scan over any inverted index of the hinted table. If no such query plan can be generated, an error is logged. [#122300][#122300]
```

Comment:
'optimizer' link should be changed to https://www.cockroachlabs.com/docs/v23.2/cost-based-optimizer

inverted index should link to https://www.cockroachlabs.com/docs/v23.2/inverted-indexes.html 

there are two spaces between the 'optimizer' link and '.. to prefer a query plan'

'error is logged' could be a link to https://www.cockroachlabs.com/docs/stable/logging-overview